### PR TITLE
fr(i18n) : Updated `nav.ts` in French language

### PR DESCRIPTION
**Pull request by thomasbnt**
- Translated content

#### Description

Updated the `nav.ts` file in French language.

### CODE REVIEW

File: src/i18n/fr/nav.ts [hunk 1]
```diff
@@ -21,32 +25,33 @@ export default NavDictionary({
 	'core-concepts/astro-components': 'Composants',
 	'core-concepts/astro-pages': 'Pages',
 	'core-concepts/layouts': 'Layouts',
-	'guides/markdown-content': 'Markdown',
+	'guides/markdown-content': 'Contenu Markdown',
```
Comment (delucis) [inline src/i18n/fr/nav.ts:28]: In English this is “Markdown & MDX” so perhaps this should match:

```suggestion
	'guides/markdown-content': 'Markdown et MDX',
```

File: src/i18n/fr/nav.ts [hunk 2]
```diff
@@ -21,32 +25,33 @@ export default NavDictionary({
...
 	'guides/imports': 'Fichiers Statiques',
 	'guides/troubleshooting': 'Dépannage',
 
 	// Features
 	features: 'Fonctionnalités',
 	'guides/configuring-astro': 'Configurer Astro',
-	'guides/styling': 'CSS et Styles',
-	'guides/data-fetching': 'Récupération de Données',
+	'guides/styling': 'CSS et Style',
+	'guides/data-fetching': 'Récupération de données',
```
Comment (delucis) [inline src/i18n/fr/nav.ts:36]: I’m noticing that many of the entries use title case, so maybe we should stick with capital D?

```suggestion
	'guides/data-fetching': 'Récupération de Données',
```

File: src/i18n/fr/nav.ts [hunk 3]
```diff
@@ -21,32 +25,33 @@ export default NavDictionary({
...
 	'guides/deploy': 'Déployer',
 	'guides/environment-variables': "Variables d'Environnement",
-	'guides/aliases': "Alias d'Importation",
+	'guides/aliases': 'Alias d’importation',
```
Comment (delucis) [inline src/i18n/fr/nav.ts:39]: Same here:

```suggestion
	'guides/aliases': 'Alias d’Importation',
```

File: src/i18n/fr/nav.ts [hunk 4]
```diff
@@ -21,32 +25,33 @@ export default NavDictionary({
...
-	'reference/api-reference': 'API d\'Exécution',
-	'reference/integrations-reference': 'API d\'Intégration',
+	'reference/api-reference': "API d'Exécution",
+	'reference/integrations-reference': "API d'Intégration",
 	'reference/adapter-reference': 'API des adaptateurs (expérimental)',
 	'core-concepts/routing': 'Règles de Routage',
 	'reference/directives-reference': 'Utilisation des Directives',
-	'guides/publish-to-npm': 'Format de Packet NPM',
-});
+	'guides/publish-to-npm': 'Publier sur NPM',
```
Comment (delucis) [inline src/i18n/fr/nav.ts:55]: In English we have “NPM Package Format” as the label, so I think the original French translation here was correct.

```suggestion
	'guides/publish-to-npm': 'Format de Packet NPM',
```

File: src/i18n/fr/nav.ts [hunk 5]
```diff
@@ -21,32 +25,33 @@ export default NavDictionary({
...
+	'guides/markdown-content': 'Contenu Markdown',
 	'guides/imports': 'Fichiers Statiques',
 	'guides/troubleshooting': 'Dépannage',
 
 	// Features
 	features: 'Fonctionnalités',
 	'guides/configuring-astro': 'Configurer Astro',
-	'guides/styling': 'CSS et Styles',
-	'guides/data-fetching': 'Récupération de Données',
+	'guides/styling': 'CSS et Style',
```
Comment (delucis) [inline src/i18n/fr/nav.ts:35]: Could we get a second francophone opinion on this @Chrissdroid @Princesseuh?

The English text is “CSS & Styling”.

**Review by delucis — COMMENTED** [review]
Thank you @thomasbnt I have some comments and would love another pair of French eyes to double check this too

**Comment by thomasbnt** [comment]
Hello and thanks for your comments @delucis, approved all reviews, after read that It's better !

**Review by delucis — APPROVED** [review]

**Comment by delucis** [comment]
Thanks again @thomasbnt! 🙌

**Pull request merged**